### PR TITLE
setup centrifugo backend

### PR DESCRIPTION
**Pull request by ALASHI1**
setup centrifugo backend

### CODE REVIEW

File: backend/deals/views.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import requests, json
 from rest_framework.response import Response
 from rest_framework import status as st
-
+from common.utils import *
```
Comment (shodown96) [inline backend/deals/views.py:8]: change this to single import

File: backend/prospect/views.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 from rest_framework import status
 
 from .serializers import ProspectSerializer
-
+from common.utils import *
```
Comment (shodown96) [inline backend/prospect/views.py:12]: same here

**Review by shodown96 — CHANGES_REQUESTED** [review]
Please take a look at my review of your PR.

File: backend/deals/views.py (hunk 1 above)
Comment (ALASHI1) [reply backend/deals/views.py]: Requested Changes have been made

File: backend/prospect/views.py (hunk 2 above)
Comment (ALASHI1) [reply backend/prospect/views.py]: Requested Changes have been made

**Pull request merged**